src/assets/scss/_variables.scss

```diff
@@ -27,6 +29,8 @@ $einBlue: #92c3e6;
 
 $yuxuanBlue: #0d7acb;
 $yuxuanBlueTransparent: #0d7acb1a;
+$viewButtonBlue: #4eabee;
```

Comment:
I see that these new color variables are the exact hex values from Figma, but I think it is better if we try and re-use our existing color scheme where possible. Could you instead use the closest similar blue we already have like the blue used on hovering over the onboarding button in the sidebar, or ask a designer?